charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -226,6 +226,10 @@ poller:
   # https://splunk.github.io/splunk-connect-for-snmp/main/configuration/poller-configuration/#define-usernamesecrets
   usernameSecrets: []
 
+  go:
+    # Enable logs from gosnmp library
+    gosnmpLogger: false
```

Comment:
probably we should add it also here https://github.com/splunk/splunk-connect-for-snmp/blob/main/charts/splunk-connect-for-snmp/templates/worker/_helpers.tpl#L81 as env variable in workers, and map this param to ENABLE_GO_LOGGER and I think we can set the default as false.

Reply:
Done